inference-engine/tests/functional/plugin/myriad/single_layer_tests/static_shape_broadcast.cpp

```diff
@@ -117,4 +118,75 @@ INSTANTIATE_TEST_CASE_P(smoke_accuracy, StaticShapeBroadcastLayerTest,
                                 ::testing::Values(CommonTestUtils::DEVICE_MYRIAD)),
                         StaticShapeBroadcastLayerTest::getTestCaseName);
 
-}  // namespace LayerTestsDefinitions
+
+class StaticShapeBidirectionalBroadcastLayerTest : public testing::WithParamInterface<StaticShapeBroadcastTestParam>,
```

Comment:
We can reuse `StaticShapeBroadcastLayerTest` but extracct the `BIDIRECTIONAL` mode specific code into separate method.

Reply:
I've put it into one class